source/robot_model/test/tests/test_model.cpp

```diff
@@ -72,18 +72,28 @@ TEST_F(RobotModelTest, TestJacobianJointNames) {
   }
 }
 
+TEST_F(RobotModelTest, TestJacobianFrameNames) {
+  state_representation::JointState dummy = state_representation::JointState(robot_name, 7);
+  state_representation::Jacobian jac = franka->compute_jacobian(dummy);
+  EXPECT_TRUE(jac.get_reference_frame() == "panda_link0");
```

Comment:
Maybe next time you could use `EXPECT_STREQ` to compare strings.

Reply:
Well not really as `EXPECT_STREQ` works only for raw c string (`char*`). I tried and got an error. Apparently you need to use `EXPECT_EQ`.

Reply:
you can use the `.c_str()` method on std strings to convert them to C-style string `char *`

Reply:
This is actually a very good point as the `EXPECT_TRUE` macro do not provide any feedback on the actual values. I have replaced with `EXPECT_EQ` and `EXPECT_GT` which are more verbose.